api/k8s/v1beta4/k8s_helpers.go

```diff
@@ -47,3 +47,62 @@ func (s *API) WaitForCluster(req *WaitForClusterRequest) (*Cluster, error) {
 	}
 	return cluster.(*Cluster), nil
 }
+
+// WaitForClusterNodesReadyRequest is used by WaitForClusterNodesReady method.
+type WaitForClusterNodesReadyRequest struct {
+	ClusterID string
+	Region    scw.Region
+	Timeout   time.Duration
```

Comment:
Maybe use a `*time.Duration` and set a default value?

Reply:
I don't get it. Why use a pointer ? The WaitForClusterRequest above in the same file does not. And there is no default value either.

Reply:
Hmm fair enough 🤔
There is one here though https://github.com/scaleway/scaleway-sdk-go/blob/086aa056023ea02beba9251624b26724a212bd90/api/instance/v1/server_utils.go#L106
I'll ask @jerome-quere for his input 😄

Reply:
OK. AFAIK the timeout will be provided here https://github.com/terraform-providers/terraform-provider-scaleway/pull/393/files#diff-a881280cfcfd6a01417309425777e2b8R80